symphony-bdk-core/src/test/java/com/symphony/bdk/core/service/datafeed/impl/DatafeedLoopV2Test.java

```diff
@@ -74,7 +74,12 @@ void setUp() throws BdkConfigException {
                 bdkConfig
         );
         this.listener = new RealTimeEventListener() {
-            @Override
+          @Override
+          public boolean isAcceptingEvent(V4Event event, String username) {
+            return true;
```

Comment:
do we have test checking that the event is not sent when isAcceptingEvent returns false?

Reply:
we already have a test in DatafeedLoopV1Test for that case. This method is under the AbstractDatafeedLoop class so I guess we don't need to have another in DatafeedLoopV2Test.